File: java/Day06/Demo01/src/com/sunbeam/Student.java
package com.sunbeam;

import java.util.Scanner;

public class Student extends Person {
	int rollno;
	double marks;

	@Override
	public void accept(Scanner sc) {
		System.out.print("Enter rollno - ");
		rollno = sc.nextInt();
		super.accept(sc);
		System.out.print("Enter marks - ");
		marks = sc.nextDouble();
	}

	@Override
	public void display() {
		System.out.println("Rollno -  " + rollno);
		super.display();
		System.out.println("Marks -  " + marks);
	}

	void calculatePercentage() {
		System.out.println("Percentage = " + ((marks / 150) * 100));
	}

}

File: cpp/Day11/attendancesystem/student.cpp
#include "student.h"

Student::Student()
{
    cout << "Student()" << endl;
}
Student::Student(double marks)
{
    cout << "Student(double)" << endl;
}
void Student::accept()
{
    cout << "Student::accept()" << endl;
}

void Student::display()
{
    cout << "Student::display()" << endl;
}